SQL-MYSQL/gestao_produtos/criar_banco_produtos.sql

```diff
@@ -0,0 +1,22 @@
+create database if not exists gestao_produtos;
+
+use gestao_produtos;
+
+create table produtos (
+    id INT AUTO_INCREMENT primary key,
+    nome VARCHAR(255) not null,
+    sku VARCHAR(255) not null,
+    descricao TEXT,
+    categoria VARCHAR(255),
+    preco DECIMAL(10, 2),
+    unidade_medida VARCHAR(50),
+    peso DECIMAL(10, 2),
+    quantidade_estoque INT,
+    fabricante VARCHAR(255),
+    fornecedor VARCHAR(255),
+    deleted_at datetime,
```

Comment:
Recomento vc forçar o valor nulo no deleted_at e já definir nulo como padrão tb:
`deleted_at TIMESTAMP NULL DEFAULT NULL,`